mkdocs.yml

```diff
@@ -0,0 +1,86 @@
+site_name: "MCAP"
```

Comment:
possibly dumb question: why does this file live in the root of repo rather than in the `docs` folder?

Reply:
I'm not perfectly clear on why this must be, but I can say that it's considered an error if you try to put this file in the `docs` folder.
```
ERROR    -  Config value: 'docs_dir'. Error: The 'docs_dir' should not be the parent directory of the config file. Use a child
            directory instead so that the 'docs_dir' is a sibling of the config file.
Aborted with 1 Configuration Errors!
```
See also https://www.mkdocs.org/user-guide/writing-your-docs/#file-layout

Reply:
😭